seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/stream/SparkStreamingExecution.scala

```diff
@@ -48,12 +48,9 @@ class SparkStreamingExecution(sparkEnvironment: SparkEnvironment)
             dataset)
         }
         var ds = dataset
-
-        if (ds.take(1).length > 0) {
```

Comment:
Can't delete this, check this: https://github.com/apache/incubator-seatunnel/pull/2578